### ISSUE TITLE
PoB really slow/bad performance

### ISSUE THREAD

**Issue by procrastin8** [PathOfBuildingCommunity/PathOfBuilding#5310]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### How is Path of Building expected to behave?

If i tab to PoB i should not have to wait for it to load for 2 mins

### How does Path of Building behave?

PoB feels extremely slow lately, like sometimes it will take 1-2 mins to render the window if I tab to it or I just navigate around or switch trees

### How to reproduce the issue

have it open for a few hours behind other stuff then try and click on the pob window in taskbar or tab to it. It will be stupidly slow

### Character build code

```shell
n/a
```

### Screenshots

n/a

**Comment by QuickStick123** [comment]
A few more details would be useful here like info on computer, os and build. As others aren't really observing it. You could try reinstalling drivers and maybe that will help. There has been a couple seconds added to start up tine recently though afaik but that shouldn't affect long term performance.

**Comment by gaborfekete-crey** [comment]
I experience the same issue, after some time (few hours)  switching back to PoB takes a lot of time, and it doesn't accept inputs for several minutes. Restarting the program does solve this slowdown but it's cumbersome to even kill it's process. I also suspect that as it is using too many resources, my other programs are slowed down too.
My setup:
Processor	Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz   2.80 GHz
Installed RAM	16.0 GB (15.9 GB usable)
System type	64-bit operating system, x64-based processor
GPU: Nvidia GeForce GTX 1060

**Comment by procrastin8** [comment]
Also I have multiple friends and have observed Crouching_Tuna experience the same issue on stream.

**Comment by knasen88** [comment]
![image](https://user-images.githubusercontent.com/16074363/208085139-69b76e47-fc20-438d-bf9d-b6efd80ef397.png)
i also have very high cpu usage (when focused). i'd kinda expect this with node power enabled but doing nothing shouldn't be eating CPU like this

**Comment by rolliono** [comment]
I experience a performance issue too since I've switched to 2.24.0. I have no lags on alt-tab per se, but the app takes about 3 seconds to respond to any kind of action like mouse click or showing info about item or node under the cursor. The issue persists both when I have a build open or when I'm in the builds folder.  

My setup: 
amd ryzen 5 2600
rx580
32Gb RAM 
x64 Win10

build example - https://pastebin.com/WQq2UPwT

**Comment by QuickStick123** [comment]
If it wouldn't trouble you guys could you try this build https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4736#issuecomment-1211478087 it is a WIP that we have been telling people to use with a certain type of crash that causes the app not to start but I am curious to see if it resolves the slow response after a while. It has some visual bugs but uses a completely different renderer which we are planning to move towards once it is a bit more tested and remaining bugs are fixed for wide release.
@rolliono @procrastin8 @gaborfekete-crey

**Comment by rolliono** [comment]
The issue resolved itself after I reinstalled my graphic drivers. Didn't even try to fix THIS, just noticed that windows Update messed with Adrenaline software and decided to repair drivers with a fresh version from amd site. Funny that))

**Comment by QuickStick123** [comment]
Yeah a lot of people have been reporting reinstalling graphics drivers is fixing there problems.

**Comment by procrastin8** [comment]
interesting, I will try reinstalling drivers

**Comment by procrastin8** [comment]
reinstalling drivers did not work

**Comment by QuickStick123** [comment]
@procrastin8 
Did you try https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/5310#issuecomment-1355950279 I am interested to see if it does anything. I don't have high hopes but if it does it is enouther reason to try and get that released sooner.

> If it wouldn't trouble you guys could you try this build [#4736 (comment)](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/4736#issuecomment-1211478087) it is a WIP that we have been telling people to use with a certain type of crash that causes the app not to start but I am curious to see if it resolves the slow response after a while. It has some visual bugs but uses a completely different renderer which we are planning to move towards once it is a bit more tested and remaining bugs are fixed for wide release.

**Comment by Nightblade** [comment]
Another data point:
> **Admirable_Scholar_41** via [reddit](https://www.reddit.com/r/pathofexile/comments/zpcqki/pob_lagging_severely_since_320_update/)
> System is laptop Asus TUF A15 with AMD Ryzen 5 processor and Nvidia graphics, 16 G graphic memory, OS is Windows 11 21H2, 64 bit. SSD hard drive.
>
>The problem is regardless of PoE (happens when PoE is off), and started with one of the recent 3.20 updates

**Comment by QuickStick123** [comment]
Get them to try an old version as well if possible
